pkg/edgehub/config/config.go

```diff
@@ -111,36 +119,39 @@ func getWebSocketConfig() error {
 	return nil
 }
 
-func getControllerConfig() {
+func getControllerConfig() error {
 	heartbeat, err := config.CONFIG.GetValue("edgehub.controller.heartbeat").ToInt()
 	if err != nil {
-		log.LOGGER.Warnf("failed to get key file for web socket client")
-		heartbeat = handshakeTimeoutDefault
+		log.LOGGER.Warnf("failed to get heartbeat  for controller client")
+		heartbeat = heartbeatDefault
 	}
 	edgeHubConfig.CtrConfig.HeartbeatPeroid = time.Duration(heartbeat) * time.Second
 
 	interval, err := config.CONFIG.GetValue("edgehub.controller.refresh-ak-sk-interval").ToInt()
 	if err != nil {
-		log.LOGGER.Warnf("failed to get key file for web socket client")
+		log.LOGGER.Warnf("failed to get refresh-ak-sk-interval for controller client")
 		interval = refreshInterval
 	}
 	edgeHubConfig.CtrConfig.RefreshInterval = time.Duration(interval) * time.Minute
 
 	projectId, err := config.CONFIG.GetValue("edgehub.controller.project-id").ToString()
 	if err != nil {
-		log.LOGGER.Warnf("failed to get project id  for web socket client")
+		log.LOGGER.Errorf("failed to get project id  for controller client: %v", err)
+		return fmt.Errorf("failed to get project id  for controller client: %v", err)
 	}
 	edgeHubConfig.CtrConfig.ProjectID = projectId
 
 	nodeId, err := config.CONFIG.GetValue("edgehub.controller.node-id").ToString()
 	if err != nil {
-		log.LOGGER.Warnf("failed to get node id  for web socket client")
+		log.LOGGER.Errorf("failed to get node id  for controller  client: %v", err)
```

Comment:
Log message should be started with an upper-case word.

Please remove unneeded white space in the message.

Reply:
Do not log error in multiple places.